reana_client/cli/workflow.py

```diff
@@ -566,8 +585,12 @@ def workflow_validate(ctx, file):
 @add_workflow_option
 @add_access_token_options
 @click.pass_context
-def workflow_stop(ctx, workflow, force_stop, access_token):
-    """Stop given workflow."""
+def workflow_stop(ctx, workflow, force_stop, access_token):  # noqa: D301
+    """Stop a running workflow.
+
```

Comment:
Add some body, e.g.

The `stop` command permits to hard-stop the running workflow process.  Note that soft-stopping of the workflow is currently not supported.  This command should be therefore used with care, only if you are absolutely sure that there is no point in continuing the running the workflow.